docs/cmake-settings.md

```diff
@@ -0,0 +1,66 @@
+# Configure CMake tools
```

Comment:
This was confusing me too. I left it because I assumed it was terminology that the cmake community was familiar with. I like your idea for CMake tools settings and will ripple that change through the various articles.

Reply:
I've updated the configure.md and cmake-settings.md files to try to separate these ideas. I have the concept of configuring CMake tools, which I've renamed CMake tool settings, and then in configure.md I tried to be more clear about the 'CMake configure process' and changed headings accordingly.